Add repro for #13306

nemanjaglumac opened the pull request:
### Status
NEEDS REVIEW

### What does this PR accomplish?
- reproduces #13306 

### How should this be manually tested?
- replace `it.skip()` with `it.only()` and run the test

### Additional notes:
- test should pass once the issue is resolved (remove `.skip`)

@flamber 
- this test correctly fails because parenthesis are indeed being removed on blur, however
- if I comment out this part...
```javascript
// cy.log("**Fails after blur in v0.36.6**");
// cy.get("[contentEditable=true]").contains(
//   "Sum([Total]) / (Sum([Product → Price]) * Average([Quantity]))",
// );
```
... and let the test run until the end, I still see that both results are `0.51`, regardless of the fact parenthesis have been removed. o.O

Commit:
Add repro for #13306

flamber commented:
Okay, then try this instead:
```
SUM(100) / SUM(10) * SUM(5)         = 50
SUM(100) / (0 + SUM(10) * SUM(5))   = 2
```

flamber commented:
> ... and let the test run until the end, I still see that both results are 0.51, regardless of the fact parenthesis have been removed. o.O

Make it fail on the parenthesis being removed. Ignore the results and hack completely.

nemanjaglumac commented:
The test correctly confirms the input is changing to: `Sum([Total]) / Sum([Product → Price]) * Average([Quantity])` **(and it fails at that point)**.

I'd say that's good repro for #13306, because it will never even come to the calculation/visualization part before the parenthesis issue is resolved. What do you say? Good enough to merge?

Commit:
Simplify test

- removed the "incorrect" and "correct" part, as well as visualization
- test should ONLY check that parenthesis are indeed being removed on blur
